apps/demo/src/app/app.module.ts

```diff
@@ -1,20 +1,16 @@
-import { NgModule } from '@angular/core';
-import { BrowserModule } from '@angular/platform-browser';
-import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-import {
-  TuiDocMainModule,
-  TUI_DOC_LOGO,
-  TUI_DOC_PAGES,
-} from '@taiga-ui/addon-doc';
-import { TuiLinkModule } from '@taiga-ui/core';
+import {NgModule} from '@angular/core';
```

Comment:
why have you decided to change current `{ entity }` style to `{entity}` style? 🤔

Reply:
Oh, I apologize. It seems that your `editorconfig` is not sufficient for WebStorm. I will fix that. But it's probably better to have `lint-staged` for the `pre-commit` hook, which will run `Prettier` to avoid such discrepancies.

Reply:
oh, never noticed we dont have it here. We need to add it for sure, just created an issue https://github.com/Tinkoff/ng-morph/issues/81

Reply:
I added a new rule for WebStorm to editorconfig, and fixed these issues